pkg/util/bandwidth/linux.go

```diff
@@ -44,6 +44,7 @@ type tcShaper struct {
 	iface string
 }
 
+// NewTCShaper makes a new tcShaper for the given interface
 func NewTCShaper(iface string) BandwidthShaper {
```

Comment:
this type was renamed above but not throughout, so this code will not compile (which is why ~ every presubmit is failing)

Reply:
`BandwidthShaper` that is.

Reply:
Thanks for noticing!
While at it other instances where BandWidthShaper might have been called, only came across this one:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

Does it make sense? Or since it's changing another package's code should I revert and stick with BandwidthShaper?

Reply:
Done.